### ISSUE TITLE
multihead attention biases

### ISSUE THREAD

**Issue by stanchiang** [hkproj/pytorch-transformer#4]
hi, in your `model.py` code, the `MultiHeadAttentionBlock` class you create a w_q via `self.w_q = nn.Linear(d_model, d_model) # Wq`

but you call nn.Linear without setting `bias=False` so by default it seems like when we do `query = self.w_q(q)` we will also be adding and training with some extra bias values which isn't in the original paper. 

is this correct?

**Comment by hkproj** [comment]
Hi @stanchiang! You're right. There should be no bias in the W matrices. The reason behind that is that is the fact that after the attention mechanism there is a normalization layer that will recenter the data (Layer Normalization recenters it, while other normalization layers like RMS do not). Recentering of the data will basically get rid of the bias. 

This is why the model could work even with the bias, because Layer Normalization was getting rid of its effect.

Thanks for pointing out and I have fixed the code.

**Comment by Hope1337** [comment]
@hkproj I think the same logic should be applied to Feedforward layer, is it?
